bot/cogs/filtering.py

```diff
@@ -26,6 +26,7 @@
     flags=re.IGNORECASE
 )
 
+SPOILER_RE = re.compile(r"(\|\|.+?\|\|)")
```

Comment:
.+? matches any character (except for line terminators)

but discord allows you to use new-line in spoiler tags.

Reply:
Thanks for the heads up, I've updated the pattern to support multi-line spoilers.